Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -89,18 +89,20 @@ public class BTShopperInsightsClient {
         }
     }
 
-    /// Call this method when the PayPal button has been successfully displayed to the buyer.
+    /// Call this method when the PayPal or Venmo button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
     /// - Parameters:
-    ///    - paymentMethodsDisplayed: Optional:  The list of available payment methods, rendered in the same order in which they are displayed i.e. ['Apple Pay', 'PayPal']
-    ///    - experiment: Optional:  A `JSONObject` passed in as a string containing details of the merchant experiment.
-    public func sendPayPalPresentedEvent(paymentMethodsDisplayed: [String?] = [], experiment: String? = nil) {
-        let paymentMethodsDisplayedString = paymentMethodsDisplayed.compactMap { $0 }.joined(separator: ", ")
+    ///     - `buttonType`: Type of button presented - PayPal, Venmo, or other
+    ///     - `presentmentDetails`:  Detailed information, including button order, experiment type, and
```

Comment:
It's simply due to the small detail I've seen where other parameter names in docstrings are in a monospace typeface. I've added a screenshot of what I see vs the above parameters without the backticks. It's not a visual difference that someone will see when comparing code diffs but I can remove for consistency

<img width="635" alt="Screenshot 2024-12-17 at 9 37 45 AM" src="https://github.com/user-attachments/assets/b5088281-dd11-4978-a807-77799e98c86a" />

<img width="635" alt="Screenshot 2024-12-17 at 9 38 16 AM" src="https://github.com/user-attachments/assets/0208b152-046c-466a-8930-8578a417c9c4" />

Reply:
Gotcha, that sounds good. I was more referencing the docs merchants see when you right click a method to "show quick help" which shows the syntax merchants would see. It looks like without the backticks is what the apple generated docs expects for the listed parameters. I am guessing the second isn't generated the same because it's indented in a little too far.

Reply:
You're right, it was indented 1 space too far. Fixed now :) 
<img width="616" alt="Screenshot 2024-12-18 at 1 09 56 PM" src="https://github.com/user-attachments/assets/00c2150e-c304-4539-9acf-2332c2367ad5" />